src/glabs-native-traffic-driver/amp/index.scss

```diff
@@ -72,8 +72,9 @@ $brand-colour: #69d1ca;
 
 .creative__brand-label {
     background: $brand-colour;
-    color: #767676;
+    color: #333;
```

Comment:
Boring comment, I know, but worth sticking this in a variable since it is used elsewhere and "magic numbers" etc...?